Update monospacifier.py

PatienceAllergy opened the pull request:
Due to a potential bug in font-forge, the right/left side glyph bearings need integers, despite error messages claiming to need floats.

PatienceAllergy committed:
Update monospacifier.py

Due to a potential bug in font-forge, the right/left side glyph bearings need integers, despite error messages claiming to need floats.

win98se commented:
Please help to add that this pull request will close #32 in your PR description, so that the issue will be automatically closed when this PR is merged. :+1:

win98se reviewed (changes requested):
Using `math.round()` returns an error that reads `AttributeError: module 'math' has no attribute 'round'`; Changing it to `round()` (without `math.`) solves it.

PatienceAllergy commented:
> Using `math.round()` returns an error that reads `AttributeError: module 'math' has no attribute 'round'`; Changing it to `round()` (without `math.`) solves it.

You're right! I tested it, and it built a font, but I didn't notice the error!

It works perfectly without the "math." Now let's see how I can update this thing...

win98se commented:
> Now let's see how I can update this thing...

Just push an update to your fork and this PR will be updated as well. ;)

PatienceAllergy committed:
no need to call "math" object to round

One again, I did a test font creation. I didn't notice any errors. Here's hoping...

PatienceAllergy commented:
> > Now let's see how I can update this thing...
> 
> Just push an update to your fork and this PR will be updated as well. ;)

It looks like somehow I did that, not sure how, but yay!
